src/main/java/theflogat/technomancy/lib/compat/ModuleBase.java

```diff
@@ -28,9 +28,11 @@ public static void registerBlock(Block block, String name) {
 		if(block instanceof BlockContainerAdvanced){
 			GameRegistry.registerBlock(block, ItemAdvancedBase.class, name);
 			return;
+		} else {
+			if(block!=null) {
+				GameRegistry.registerBlock(block, name);
+			}
 		}
```

Comment:
OOPS. I didn't mean to commit this. It won't hurt anything, but it's also not really needed.